docs/import-data/csv/load-csv-clause.md

```diff
@@ -63,8 +70,8 @@ files into the container.
 
 Below, you can find two examples of how to use the LOAD CSV clause depending on
 the complexity of your data:
-- [Transferring CSV files into a Docker container](#transfer)
-- Examples
+- [Transferring CSV files into a Docker container {#transfer}](#transferring-csv-files-into-a-docker-container-transfer)
```

Comment:
Fix the {#transfer}, it does not render well

Reply:
this is some kind of new VSC? feature ... it makes tables of content out of everything... idiot